inventory/group_vars/all.yaml

```diff
@@ -13,19 +14,19 @@ fcos_image_name: fedora-coreos-31.20200118.3.0-qemu.x86_64.qcow2
 binaries_location: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/
 
 workdir:
-  vm_data: /home/rspazzol/git/ocp4-ansible-libvirt/vms
-  install: /home/rspazzol/git/ocp4-ansible-libvirt/install
-  registry_mirror: /home/rspazzol/git/ocp4-ansible-libvirt/registry-mirror
+  vm_data: "/home/{{ homedir_user }}/git/ocp4-ansible-libvirt/vms"
+  install: "/home/{{ homedir_user }}/git/ocp4-ansible-libvirt/install"
+  registry_mirror: "/home/{{ homedir_user }}/git/ocp4-ansible-libvirt/registry-mirror"
 
 mirror:
-  username: raffa
-  password: raffa
+  username: "raffa"
+  password: "raffa"
   pull_secret_file: pull_secret.json
   sigstore: https://mirror.openshift.com/pub/openshift-v4/signatures/
   tang_secret: PLjNyRdGw03zlRoGjQYMahSZGu9
 
 cluster:
-  version: 4.3.3
```

Comment:
why this change?

Reply:
Accidental left over config. I was thinking maybe the verison could be moved to the .env file and be set there? e.g. OC_CLUSTER_VERSION?